### PR TITLE
Move demo app into new examples/ directory

### PR DESCRIPTION
**Pull request by developit**

### CODE REVIEW

File: examples/demo/package.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+	"name": "@examples/demo",
+	"version": "0.0.0",
+	"private": true,
+	"type": "module",
+	"alias": {
+		"react": "preact/compat"
+	},
+	"devDependencies": {
+		"wmr": "*"
```
Comment (marvinhagemeister) [inline examples/demo/package.json:10]: We should probably move/update the `demo` script in wmr's `package.json` too

**Comment by marvinhagemeister** [comment]
Looks like some paths in the CI need to be updated too.

File: package.json [hunk 2]
```diff
@@ -3,11 +3,14 @@
...
-			"./packages/*"
+			"packages/*",
+			"examples/*"
 		]
 	},
 	"scripts": {
-		"ci": "yarn workspace wmr ci"
+		"demo": "yarn workspace @examples/demo run",
+		"wmr": "yarn workspace wmr run",
+		"ci": "yarn wmr build && yarn demo build:prod"
```
Comment (developit) [inline package.json:13]:
```suggestion
		"ci": "yarn wmr build && yarn && yarn demo build:prod"
```

File: README.md [hunk 3]
```diff
@@ -59,10 +59,10 @@ cd wmr
 yarn
 
 # run the demo (no compile)
-yarn workspace wmr demo
+yarn demo serve
 
 # build and serve the demo for prod
-yarn workspace wmr demo:prod && yarn workspace wmr demo:serve
+yarn demo build:prod && yarn demo serve
```
Comment (developit) [inline README.md:65]:
```suggestion
yarn demo build:prod && yarn demo serve:prod
```

**Review by marvinhagemeister — APPROVED** [review]

**Pull request merged**